package.json

```diff
@@ -199,14 +199,14 @@
     "docs": "sf-docs",
     "format": "sf-format",
     "lint": "eslint \"src/**/*.ts\"",
-    "oclif-artifacts": "oclif manifest",
+    "oclif-artifacts": "oclif manifest .",
```

Comment:
please explain this one to me.

Reply:
@mshanemc I believe the command will hang on windows if the dot is not present.

Reply:
This is all the explanation I got: https://salesforce-internal.slack.com/archives/G02K6C90RBJ/p1635790670258600?thread_ts=1635183450.084400&cid=G02K6C90RBJ

Builds were hanging on that step without it, working with it. But not on Windows, on the CircleCI Linux machines.